### DISCUSSION TITLE
<SPC>f vs. <SPC>F

### DISCUSSION THREAD

**Discussion by fominok** [helix-editor/helix#11979] [Q&A] [answered]
### Summary

Both claim to open a file picker but with the `F` version to open it "in current working directory", however, both do the same; shall they do something different?

### Reproduction Steps

 

I tried this:

1. Opened a file
2. Changed the working dir
3. Tried both `<SPC>f` and `<SPC>F`

I expected this to happen:

Any difference in behavior

Instead, this happened:

Both pickers showed me contents of the directory I did `:cd` to 

### Platform

GNU/Linux

### Terminal Emulator

alacritty

### Installation Method

pacman

### Helix Version

24.7

**Comment by sicher** [comment]
`<spc>f` opens picker in the root, where you started helix.

**Comment by fominok** [comment]
> `<spc>f` opens picker in the root, where you started helix.

I'm afraid it's not, with `:cd` both `f` and `F` show me the new dir contents

**Comment by the-mikedavis** [comment, marked as answer]
`<space>F` always opens in the current working directory while `<space>f` attempts to find your workspace root (ancestor directory with `.git`, `.svn`, `.jj` or `.helix` currently). For an example of where they are different, if you start helix in a subdirectory of a repository like `docs/` in this repository, `<space>f` opens in the repository root while `<space>F` opens in `docs/`.

**Reply by fominok** [reply]
Thanks, now I can see the difference
